src/components/common/contact-form.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../scss/colors.scss';
+@import '../../scss/breakpoints.scss';
+
+.contact-form {
+  margin-top: 30px;
+
+  input[type='text'],
+  input[type='email'],
+  select,
+  textarea {
+    width: 50%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+    margin-top: 6px;
+    margin-bottom: 16px;
+    resize: vertical;
+    margin-bottom: 50px;
+  }
+
+  @media (max-width: $viewport-md) {
+    input[type='text'],
+    input[type='email'],
+    select,
+    textarea {
+      width: 100%;
+    }
+  }
+
+  .input-header {
+    margin-bottom: 5px;
+    font-weight: 600;
+    font-size: 24px;
+  }
+
+  button {
+    width: 210px;
+    height: 58px;
+    background: $color-blueberry-200;
+    border-radius: 4px;
+    line-height: 33px;
+    color: $color-infobox-sync-dark;
```

Comment:
I would opt for one of the blueberry/plum colors here. (The infobox colors are not _really_ part of the palette -- that's my bad for causing that confusion.) You might be able to use the existing `Button` component with the form, too.

Reply:
Ooh, reusing that component is a good idea.

It looks like ESLint doesn't want React button types to be dynamically defined, so unfortunately I don't believe I can just go `type={buttonType}`. Does the below seem reasonable? (Should it include the `reset` case too...?) Trying not break anything that already uses Button.

```
const Button = ({ big, children, type = 'button' }) => {
  if (type === 'submit') {
    return (
      <button className={`button${big ? ' button-big' : ''}`} type="submit">
        {children}
      </button>
    )
  }
  return (
    <button className={`button${big ? ' button-big' : ''}`} type="button">
      {children}
    </button>
  )
}```

Reply:
Updating the component is fine with me.